index.js

```diff
@@ -0,0 +1,15 @@
+var jwtHook = require('./lib/app.js');
+
+module.exports = function (sails) {
+    jwtHook.adaptSails(sails);
+
+    return {
+      defaults: {},
+
+      initialize: function(cb) {
+        jwtHook.init(sails);
+
+        return cb();
+      }
+    };
+};
```

Comment:
For all files:

- Newline at eof
- Use spaces in stead of tabs
- Use new var statement in stead of comma separated